pages/deregDone.tsx

```diff
@@ -0,0 +1,10 @@
+export default function deregDone(){
+    return(
+        <div className="container">
+            <div className="jumbotron">
+            <h1 className="display-4">Success</h1>
+            <p>Your locker reservation has been removed. Thank you for using our service</p>
+            </div>
+        </div>
+    );
+}
```

Comment:
Perhaps using a cleaner notation is better here?
`/unregistration/success`? `deregdone` is kinda awkward.